packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -24,6 +24,26 @@ const cli = {
         describe: 'A list of themes for the test run.',
         default: ['terra-default-theme'],
       },
+      hostname: {
+        type: 'string',
+        describe: 'Automation driver host address.',
```

Comment:
This is http://0.0.0.0 by default for the Launcher right?

Reply:
The Launcher/WebdriverIO config default is `localhost`:

https://webdriver.io/docs/options.html

https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/config/wdio.conf.js#L109

Reply:
Should we document this so it's not hidden knowledge? I assume we didn't add defaults so the launcher defaults will be used

Reply:
I think this would be fine to document in the table on the doc page. We'll want to be careful with documentation since these defaults get applied at different areas in the tech stack.

The CLI has no defaults for these values since they get applied internally by the launcher and WDIO config.

Reply:
Why do you think we need to be careful about communication the behavior of the script? Consumers won't and shouldn't know or care of the implementation details of the script. If we are worried about being too reliant on the defaults changing, the CLI should provide it's own defaults IMO.

Reply:
We need to be careful because these defaults are not applied at this level. There is not actually a default at this level, but there is a default hostname that is applied through the tech stack. Documenting this behavior is fine and more thorough information can be added to our doc pages, but I don't think we should try to squeeze it all into the cli descriptions.